scipy/optimize/_highs/cython/src/HConst.pxd

```diff
@@ -58,7 +62,16 @@ cdef extern from "HConst.h" nogil:
         PrimalDualStatusSTATUS_MAX "PrimalDualStatus::STATUS_MAX" = PrimalDualStatusSTATUS_FEASIBLE_POINT
 
     cdef enum HighsOptionType:
-        HighsOptionTypeBOOL "HighsOptionType::BOOL" = 0
-        HighsOptionTypeINT "HighsOptionType::INT"
-        HighsOptionTypeDOUBLE "HighsOptionType::DOUBLE"
-        HighsOptionTypeSTRING "HighsOptionType::STRING"
+        HighsOptionTypeBOOL "HighsOptionType::kBool" = 0
+        HighsOptionTypeINT "HighsOptionType::kInt"
+        HighsOptionTypeDOUBLE "HighsOptionType::kDouble"
+        HighsOptionTypeSTRING "HighsOptionType::kString"
+
+    ctypedef enum ObjSense:
```

Comment:
Some new Enums like this `ObjSense` were added and are now used in `_linprog_highs.py` instead of hard-coded integers